### PR TITLE
#8670 Improve clamp to ground label

### PR DESCRIPTION
**Pull request by tdipisa**
## Description

Improve clamp to ground label in 3D style editor.

**Please check if the PR fulfills these requirements**
- [X] The commit message follows our guidelines: https://github.com/geosolutions-it/MapStore2/blob/master/CONTRIBUTING.md
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x", remove the others)
 - [ ] Bugfix
 - [ ] Feature
 - [ ] Code style update (formatting, local variables)
 - [ ] Refactoring (no functional changes, no api changes)
 - [ ] Build related changes
 - [ ] CI related changes
 - [X] Other... Please describe: translations update

## Issue
**What is the current behavior?**

#8670 

**What is the new behavior?**

## Breaking change
**Does this PR introduce a breaking change?** (check one with "x", remove the other)
 - [ ] Yes, and I documented them in migration notes
 - [x] No

### CODE REVIEW

File: web/client/translations/data.es-ES.json [hunk 1]
```diff
@@ -15,7 +15,7 @@
         "about_h20": "Licencia",
         "about_p3": "MapStore es un software libre y de código abiertas, basado en OpenLayers, Leaflet y ReactJS, y está  disponible bajo licencia BSD.",
         "about_p5-0": "Para más información, consulte",
-        "about_a0": "esta",
+        "about_a0": "esta",Klemmen Sie den Umriss an den Boden
```
Comment (allyoucanmap) [inline web/client/translations/data.es-ES.json:18]:
```suggestion
        "about_a0": "esta",
```

**Review by allyoucanmap — CHANGES_REQUESTED** [review]
- [x] If we modify this label it will change also the one for the line symbolizer so probably we need to create two different labels:

- Fill -> Clamp outline to ground
- Line -> Clamp to ground

Current Fill and line symbolizer
image

- [ ] It's also missing the Italian translation

**Review by allyoucanmap — APPROVED** [review]

**Comment by tdipisa** [comment]
@ElenaGallo please test this in DEV and let us know if we can backport to 2022.02.xx

**Comment by ElenaGallo** [comment]
Test passed on DEV, @allyoucanmap please backport to 2022.02.xx

**Pull request merged**